packages/grafana-ui/src/components/uPlot/MicroPlot.tsx

```diff
@@ -11,15 +12,17 @@ import {
   TimeZone,
   RawTimeRange,
   rangeUtil,
+  transformDataFrame,
 } from '@grafana/data';
 
 import uPlot from 'uplot';
 import { colors } from '../../utils';
 import { Themeable } from '../../types';
 import { GraphCustomFieldConfig } from './types';
+import { renderPlugin } from './renderPlugin';
 
 interface Props extends Themeable {
-  data: DataFrame; // assume applyFieldOverrides has been set
+  data: DataFrame[]; // assume applyFieldOverrides has been set
```

Comment:
@dprokop -- since uplot will only ever work with a single frame, i think we should do the merge outside (in the panel component -- not the more general graph component)